modules/core/src/main/java/org/apache/synapse/mediators/MediatorLog.java

```diff
@@ -114,8 +121,10 @@ public void auditLog(Object msg) {
      * Log a message at level DEBUG to all available/enabled logs.
      */
     public void auditDebug(Object msg) {
-        defaultLog.debug(msg);
-        if (synCtx.getServiceLog() != null) {
+    	if (defaultLog.isDebugEnabled()) { 
```

Comment:
Actually it is needed. The  other change in this pull request is to change the MediatorLog isDebugEnabled function to return true if either defaultLog or the Service Log is enabled. When we get here, at least one of the two logs are enabled, but we do not know that both are enabled. So I used the isDebugEnabled function to keep from calling the debug function of the org.apache.commins.Log object, as is standard practice.